docs/installation.rst

```diff
@@ -1,9 +1,42 @@
 Installation
 ------------
 
-Current ethPM CLI is only available via Pypi.
+Pypi (recommended)
+~~~~~~~~~~~~~~~~~~
 
-Pypi
-~~~~
+- Create your virtual environment
+- ``pip install ethpm-cli``
 
-``pip install ethpm-cli``
+
+Docker
+~~~~~~
+
+- ``docker pull ethpm/ethpm:latest``
+- ``docker run ethpm/ethpm:latest``
+
+Homebrew
```

Comment:
Any reason why this one isn't the recommended one?

Reply:
None at all - just went with pypi to encourage using more python things. Do you think Homebrew should be recommended?

Reply:
I think so.  It makes it less of a python tool and more of a general purpose tool which I think is a good thing, and it installs into the global namespace rather than whatever venv they have active (or at least it should...?).

Thoughts?  Ideally, our brew install would install itself into it's own venv (does it do that?)

Reply:
similarly, it would be nice to get an `apt-get` install path available soon too for all the linux folks.

Reply:
> (does it do that?)
yup!

Added an `apt-get` install path to issue #31